mitodl/ocw-hugo-themes
# Remove some old env vars, scripts

See https://github.com/mitodl/ocw-hugo-themes/pull/941/files#r1011986035

#### Pre-Flight checklist


#### What are the relevant tickets?
This came up when working on #941, but decided to do it as a separate PR

#### What's this PR do?
This PR removes some unused env variables and an unused script.

#### How should this be manually tested?
1. Run`yarn start:course`, inspect the output
2. Run `yarn start:www` inspect the out
3. Rune `yarn build.sh /path/to/site /path/to/config.yaml`, inspect the output

☝️ should all work fine.
